_posts/2019-04-09-PythonChallenge-Part6.md

```diff
@@ -0,0 +1,103 @@
+---
+title:  "Python Challenge, Part VI"
+date:   2019-04-09 21:00:00
+layout: post
+---
+
+<img src="/images/PyChallenge.jpg" alt="Python Challenge" align="left" hspace="5" style="width:180px;">
+
+
+Continuing on our journey through the challenges at [PythonChallenge.com](www.pythonchallenge.com), here are the next 
```

Comment:
`Continuing on our journey through the challenges at [PythonChallenge.com](www.pythonchallenge.com), here is the next`